src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -621,11 +624,29 @@ private function step_in_body() {
 			 * > "fieldset", "figcaption", "figure", "footer", "header", "hgroup",
 			 * > "main", "menu", "nav", "ol", "p", "search", "section", "summary", "ul"
 			 */
+			case '+ADDRESS':
+			case '+ARTICLE':
+			case '+ASIDE':
 			case '+BLOCKQUOTE':
+			case '+CENTER':
+			case '+DETAILS':
+			case '+DIALOG':
+			case '+DIR':
+			case '+DL':
 			case '+DIV':
```

Comment:
Nitpick: 🔤 order? (The `-` prefixed list also has `DIV` before `DL`.)

Reply:
Thanks; I've fixed the ordering.